presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -56,6 +56,12 @@ public final void destroy()
         mysqlServer = null;
     }
 
+    @Override
```

Comment:
Just thinking. 
Maybe instead of having N `supports*` methods. We could have 
`EnumSet<Feature> supportedFeatures()`

Reply:
how would implementation in a subclass look then? a switch?

Reply:
I guess superclass would be
```
return Features.all()
```

and in subclass
```
return Features.allBut(COMMENT_ON_TABLE, COMMENT_ON_COLUMN, ....)
```